stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_media_attachments.xml

```diff
@@ -21,7 +22,7 @@
         android:padding="1dp"
         app:layout_constraintEnd_toEndOf="@+id/marginEnd"
         app:layout_constraintStart_toEndOf="@+id/avatarView"
-        app:layout_constraintTop_toBottomOf="@id/gapView"
+        app:layout_constraintTop_toTopOf="parent"
```

Comment:
Why GapView is useless now? It supposes to be at the top of messages, giving additional space. Now it constrained to MediaAttachmentsGroupView and do not nothing